### DISCUSSION TITLE
PowerPlay.tflite where is this file ?

### DISCUSSION THREAD

**Discussion by Shalabh08Tayal** [FIRST-Tech-Challenge/FtcRobotController#357] [Q&A] [answered]
Hello,
We downloaded the latest code from FTCRobotController, 
Tensorflow sample code is referring to PowerPlay.tflite file, but unable to locate this file!
  private static final String TFOD_MODEL_ASSET = "c;

where can i find this file?

**Comment by texasdiaz** [comment, marked as answer]
When you compile the source code for the first time, Gradle will download the asset during compilation; it is not directly included in the FtcRobotController source.
